File: Java/Set and Get/Nomina.java
/*
 * Esta clase es la que va a llevar el constructor, con sus atributos 
 * y la invocacion de los mismos
 */
package obteneryasignar;
/**
 * 
 * @author Harold Pedraza
 * @version 16 de noviembre del 2017
 */

public class Nomina {
    //crear atributos
    String nombres, apellidos;
    int cedula, salario;
    
    //creando un metodo constructor
    
    Nomina(String nombre, String apellido, int cedula, int salario){
        //inicializando los atributos
        this.nombres = nombre ;
        this.apellidos = apellido;
        this.cedula = cedula ;
        this.salario = salario;
    }
    

    
    void Datos(){
        System.out.println(" ");
        System.out.println("El empleado es ");
        System.out.println(" ");
        System.out.println("* El nombre del empleado es: "+nombres);
        System.out.println("* El apellido del empleado es: "+apellidos);
        System.out.println("* La cedula del empleado es: "+cedula);
        System.out.println("* El salario del empleado es: "+salario);
    }
    
    String getnombres(){
        return this.nombres; 
    }
    
    String getapellidos(){
        return this.apellidos;
    }
    
    int getcedula(){
        return this.cedula;
    }
    
    int getsalario(){
        return this.salario;
    }
    
    //metodos set
    void setnombres(String NOM){
        this.nombres = NOM;
    }
    
    void setapellidos(String APE){
        this.apellidos = APE;
    }
    
    void setcedula(int CEDU){
        this.cedula = CEDU;
    }
    
    void setsalario(int SALA){
        this.salario = SALA;
    }
}

File: Python/Set and Get/Nomina.py
#Esta clase es la que va a llevar el constructor, con sus atributos y la invocacion de los mismos

__author__="Harold Pedraza"

class Nomina:

    def __init__(self, nombre, apellido, cedula, salario):
        self.nombre = nombre
        self.apellido = apellido
        self.cedula = cedula
        self.salario = salario

    def datos(self):
        print(" ")
        print("El empleado es: ")
        print(" ")
        print("El nombre del empleado es: ",self.nombre)
        print("El apellido del empleado es: ", self.apellido)
        print("La cedula del empleado es: ", self.cedula)
        print("El salario del empleado es: ", self.salario)

    """Meotodos get and set """
    def get_nombre(self):
        return self.nombre

    def get_apellido(self):
        return self.apellido

    def get_cedula(self):
        return self.cedula

    def get_salario(self):
        return self.salario


    def set_nombre(self,nom):
        self.nombre = nom

    def set_apellido(self,ape):
        self.apellido = ape

    def set_cedula(self,cedu):
        self.cedula = cedu

    def set_salario(self,sala):
        self.salario = sala